src/nvim/window.c

```diff
@@ -2797,7 +2804,7 @@ winframe_remove (
       }
     }
     frame_new_height(frp2, frp2->fr_height + frp_close->fr_height,
-        frp2 == frp_close->fr_next ? TRUE : FALSE, FALSE);
+                     frp2 == frp_close->fr_next ? true : false, false);
```

Comment:
Please please pleeease...
```suggestion
                     frp2 == frp_close->fr_next, false);
```

Reply:
Haha yeah, I also reacted on that but didn't want to touch it.